id is:3ax5y0eq
Launching wandb...
/home/tho/anaconda3/envs/scrimp/lib/python3.7/site-packages/tensorflow/python/framework/dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/tho/anaconda3/envs/scrimp/lib/python3.7/site-packages/tensorflow/python/framework/dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/home/tho/anaconda3/envs/scrimp/lib/python3.7/site-packages/tensorflow/python/framework/dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/home/tho/anaconda3/envs/scrimp/lib/python3.7/site-packages/tensorflow/python/framework/dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/home/tho/anaconda3/envs/scrimp/lib/python3.7/site-packages/tensorflow/python/framework/dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/home/tho/anaconda3/envs/scrimp/lib/python3.7/site-packages/tensorflow/python/framework/dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
Launching tensorboard...
Logging txt...
/home/tho/anaconda3/envs/scrimp/lib/python3.7/site-packages/torch/cuda/amp/grad_scaler.py:115: UserWarning: torch.cuda.amp.GradScaler is enabled, but CUDA is not available.  Disabling.
  warnings.warn("torch.cuda.amp.GradScaler is enabled, but CUDA is not available.  Disabling.")
[36m(pid=25266)[39m /home/tho/anaconda3/envs/scrimp/lib/python3.7/site-packages/torch/amp/autocast_mode.py:198: UserWarning: User provided device_type of 'cuda', but CUDA is not available. Disabling
[36m(pid=25266)[39m   warnings.warn('User provided device_type of \'cuda\', but CUDA is not available. Disabling')
[36m(pid=25261)[39m /home/tho/anaconda3/envs/scrimp/lib/python3.7/site-packages/torch/amp/autocast_mode.py:198: UserWarning: User provided device_type of 'cuda', but CUDA is not available. Disabling
[36m(pid=25261)[39m   warnings.warn('User provided device_type of \'cuda\', but CUDA is not available. Disabling')
[36m(pid=25259)[39m /home/tho/anaconda3/envs/scrimp/lib/python3.7/site-packages/torch/amp/autocast_mode.py:198: UserWarning: User provided device_type of 'cuda', but CUDA is not available. Disabling
[36m(pid=25259)[39m   warnings.warn('User provided device_type of \'cuda\', but CUDA is not available. Disabling')
[36m(pid=25264)[39m /home/tho/anaconda3/envs/scrimp/lib/python3.7/site-packages/torch/amp/autocast_mode.py:198: UserWarning: User provided device_type of 'cuda', but CUDA is not available. Disabling
[36m(pid=25264)[39m   warnings.warn('User provided device_type of \'cuda\', but CUDA is not available. Disabling')
[36m(pid=25262)[39m /home/tho/anaconda3/envs/scrimp/lib/python3.7/site-packages/torch/amp/autocast_mode.py:198: UserWarning: User provided device_type of 'cuda', but CUDA is not available. Disabling
[36m(pid=25262)[39m   warnings.warn('User provided device_type of \'cuda\', but CUDA is not available. Disabling')
[36m(pid=25265)[39m /home/tho/anaconda3/envs/scrimp/lib/python3.7/site-packages/torch/amp/autocast_mode.py:198: UserWarning: User provided device_type of 'cuda', but CUDA is not available. Disabling
[36m(pid=25265)[39m   warnings.warn('User provided device_type of \'cuda\', but CUDA is not available. Disabling')
[36m(pid=25263)[39m /home/tho/anaconda3/envs/scrimp/lib/python3.7/site-packages/torch/amp/autocast_mode.py:198: UserWarning: User provided device_type of 'cuda', but CUDA is not available. Disabling
[36m(pid=25263)[39m   warnings.warn('User provided device_type of \'cuda\', but CUDA is not available. Disabling')
[36m(pid=25260)[39m /home/tho/anaconda3/envs/scrimp/lib/python3.7/site-packages/torch/amp/autocast_mode.py:198: UserWarning: User provided device_type of 'cuda', but CUDA is not available. Disabling
[36m(pid=25260)[39m   warnings.warn('User provided device_type of \'cuda\', but CUDA is not available. Disabling')
[36m[1m(scheduler +44s)[39m[22m Tip: use `ray status` to view detailed cluster status. To disable these messages, set RAY_SCHEDULER_EVENTS=0.
[33m[1m(scheduler +44s)[39m[22m Warning: The following resource request cannot be scheduled right now: {'GPU': 0.0588, 'CPU': 1.0}. This is likely due to all cluster resources being claimed by actors. Consider creating fewer actors or adding more nodes to this Ray cluster.
2024-03-07 01:15:27,223	WARNING worker.py:1228 -- The actor or task with ID ffffffffffffffffd79f521161a2b8e98a11d20201000000 cannot be scheduled right now. You can ignore this message if this Ray cluster is expected to auto-scale or if you specified a runtime_env for this actor or task, which may take time to install.  Otherwise, this is likely due to all cluster resources being claimed by actors. To resolve the issue, consider creating fewer actors or increasing the resources available to this Ray cluster.
Required resources for this actor or task: {GPU: 0.058800}, {CPU: 1.000000}
Available resources on this node: {0.000000/8.000000 CPU, 336238680.029297 GiB/336238680.029297 GiB memory, 0.529600/1.000000 GPU, 168119339.990234 GiB/168119339.990234 GiB object_store_memory, 1.000000/1.000000 node:192.168.1.209}
In total there are 0 pending tasks and 8 pending actors on this node.
[33m[1m(scheduler +1m19s)[39m[22m Warning: The following resource request cannot be scheduled right now: {'GPU': 0.0588, 'CPU': 1.0}. This is likely due to all cluster resources being claimed by actors. Consider creating fewer actors or adding more nodes to this Ray cluster.
[33m[1m(scheduler +1m54s)[39m[22m Warning: The following resource request cannot be scheduled right now: {'GPU': 0.0588, 'CPU': 1.0}. This is likely due to all cluster resources being claimed by actors. Consider creating fewer actors or adding more nodes to this Ray cluster.
[33m[1m(scheduler +2m29s)[39m[22m Warning: The following resource request cannot be scheduled right now: {'GPU': 0.0588, 'CPU': 1.0}. This is likely due to all cluster resources being claimed by actors. Consider creating fewer actors or adding more nodes to this Ray cluster.
[33m[1m(scheduler +3m4s)[39m[22m Warning: The following resource request cannot be scheduled right now: {'CPU': 1.0, 'GPU': 0.0588}. This is likely due to all cluster resources being claimed by actors. Consider creating fewer actors or adding more nodes to this Ray cluster.
[33m[1m(scheduler +3m39s)[39m[22m Warning: The following resource request cannot be scheduled right now: {'CPU': 1.0, 'GPU': 0.0588}. This is likely due to all cluster resources being claimed by actors. Consider creating fewer actors or adding more nodes to this Ray cluster.
[33m[1m(scheduler +4m14s)[39m[22m Warning: The following resource request cannot be scheduled right now: {'GPU': 0.0588, 'CPU': 1.0}. This is likely due to all cluster resources being claimed by actors. Consider creating fewer actors or adding more nodes to this Ray cluster.
[33m[1m(scheduler +4m49s)[39m[22m Warning: The following resource request cannot be scheduled right now: {'CPU': 1.0, 'GPU': 0.0588}. This is likely due to all cluster resources being claimed by actors. Consider creating fewer actors or adding more nodes to this Ray cluster.
[33m[1m(scheduler +5m25s)[39m[22m Warning: The following resource request cannot be scheduled right now: {'CPU': 1.0, 'GPU': 0.0588}. This is likely due to all cluster resources being claimed by actors. Consider creating fewer actors or adding more nodes to this Ray cluster.
[33m[1m(scheduler +6m0s)[39m[22m Warning: The following resource request cannot be scheduled right now: {'GPU': 0.0588, 'CPU': 1.0}. This is likely due to all cluster resources being claimed by actors. Consider creating fewer actors or adding more nodes to this Ray cluster.
CTRL-C pressed. killing remote workers
Saving Final Model !
Traceback (most recent call last):
  File "/home/tho/SCRIMP/driver.py", line 388, in <module>
    main()
  File "/home/tho/SCRIMP/driver.py", line 294, in main
    "reward": eval_performance_dict['per_r']}
UnboundLocalError: local variable 'eval_performance_dict' referenced before assignment
Traceback (most recent call last):
  File "/home/tho/SCRIMP/driver.py", line 388, in <module>
    main()
  File "/home/tho/SCRIMP/driver.py", line 294, in main
    "reward": eval_performance_dict['per_r']}
UnboundLocalError: local variable 'eval_performance_dict' referenced before assignment